package lotto.domain;

class UserLottoTest {

//    @Test
//    void 로또_넘버가_6개를_넘으면_예외처리() {
//        assertThatExceptionOfType(IllegalArgumentException.class)
//            .isThrownBy(() -> new UserLotto(1, 2, 3, 4, 5, 6, 7))
//            .withMessage("[ERROR] 로또 넘버는 6개여야 합니다.");
//    }
//
//    @Test
//    void 로또_넘버에_중복이_있으면_예외처리() {
//        assertThatExceptionOfType(IllegalArgumentException.class)
//            .isThrownBy(() -> new UserLotto(1, 2, 4, 4, 5, 5))
//            .withMessage("[ERROR] 로또 넘버에 중복이 있습니다.");
//    }
}
